api/core/services/customers/customer_service.py

```diff
@@ -109,7 +109,7 @@ async def _on_channel_created(self, channel_id: str, slug: str, org_id: str | No
 
             if user:
                 assessment = await CustomerAssessmentService.run_customer_assessment(user.email)
-                await clt.set_channel_topic(channel_id, str(assessment))
```

Comment:
We now send a message because the client assessment is too long to be a topic or description.